utils/common/atomicops.h

```diff
@@ -165,6 +165,28 @@ inline bool atomicCAS(volatile T* mem, T comp, T swap)
 #endif
 }
 
+// implements a zero out of a variable
+template <typename T>
+inline void atomicZero(volatile T* mem)
+{
+#ifdef _MSC_VER
```

Comment:
I am still sure we won't support windows anymore so it is better to avoid adding new preprocessor Windows related conditions.

Reply:
I added this as following the pattern of all the other functions in the lib. Don't assume never. Never tends to byte programmers in the butt.

Reply:
Hm, all others are removing these blocks but I won't force you removing this particular block.